MathCalc.ts

```diff
@@ -0,0 +1,28 @@
+//Union Types
+type AlphaNumbers= (number | string);
+
+let AddConcatNumStr:(x:AlphaNumbers,y:AlphaNumbers)=>AlphaNumbers;
+
+AddConcatNumStr=(x:AlphaNumbers,y:AlphaNumbers)=>
```

Comment:
Here, `AddConcatNumStr` is a variable that contains a function. Use lower case names for variables and use either `const` or `let`: 

```
const addConcatNumStr = (x: AlphaNumbers, y: AlphaNumbers) => {
```

See how the spaces make the code more readable?

Reply:
I have corrected the variable to "addConcatNumStr "and also resolved the spacing issue.
I have run "npm initi -y" and installed "eslint" package in the project to check issues in typescript files using command "npm run lint"